lib/modules/apostrophe-attachments/lib/api.js

```diff
@@ -115,13 +118,29 @@ module.exports = function(self, options) {
             } else {
               info.portrait = true;
             }
+
+            if (info.extension === 'jpg' || info.extension === 'jpeg') {
+              const data = exif.parseSync(file.path);
```

Comment:
Although it is annoying, you should use exif.parse, not exif.parseSync. I read the source and it loads the whole dern JPEG into a buffer. That's tolerable but not if we do it sync, forcing all other web requests to wait for it.